### PR TITLE
Dockerize isolate server

### PR DESCRIPTION
**Pull request by mederka**
Adds a Dockerfile for making it easy to build a server container

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ENV POETRY_VENV=/opt/poetry-venv
+ENV POETRY_CACHE_DIR=/opt/.cache
+
+# Install poetry separated from system interpreter
+RUN python3 -m venv $POETRY_VENV \
+    && $POETRY_VENV/bin/pip install -U pip setuptools \
+    && $POETRY_VENV/bin/pip install poetry==${POETRY_VERSION}
+
+# Add `poetry` to PATH
+ENV PATH="${PATH}:${POETRY_VENV}/bin"
```
Comment (chamini2) [inline Dockerfile:17]: 👏

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+# Install dependencies
+COPY poetry.lock poetry.lock
+COPY pyproject.toml pyproject.toml
+RUN poetry install && poetry install -E server
```
Comment (chamini2) [inline Dockerfile:22]: Is `poetry install -E server` enough? or you need to run it twice?

**Review by chamini2 — APPROVED** [review]
very cool

File: src/isolate/server/_wsgi.py [hunk 3]
```diff
@@ -0,0 +1,4 @@
+from isolate.server import app
+
+if __name__ == "__main__":
+    app.run()
```
Comment (turbo1912) [inline src/isolate/server/_wsgi.py:4]: do you need this file I think it works if you point a folder "src/isolate/server", but this is with running `flask --app src/isolate/server/`

Comment (mederka) [reply src/isolate/server/_wsgi.py:4]: I think for now we can remove this file and just use a `flask` process.

File: Dockerfile (hunk 2 above)
Comment (mederka) [reply Dockerfile:22]: Turns out it's enough.

**Pull request merged**